lib/pbench/cli/server/report.py

```diff
@@ -149,7 +207,7 @@ def report_cache(tree: CacheManager):
     oldest_cache_name = None
     newest_cache = None
     newest_cache_name = None
-    smallest_cache = 0
+    smallest_cache = GINORMOUS
```

Comment:
You don't want to pull the "ginormous" trick with `oldest_cache` (and the zero trick with `newest_cache`?)